content/doc/developer/publishing/plugin-site.adoc

```diff
@@ -1,8 +1,109 @@
 ---
 title: Plugin Site
 layout: developersection
-wip: true
-references:
-- url: https://wiki.jenkins.io/display/JENKINS/Plugin+Site+Information+for+Plugin+Developers
-  title: Plugin site information on the wiki
 ---
+
+This page explains how https://plugins.jenkins.io/[the plugins site]
+works. The intended audience is plugin developers.
```

Comment:
Please use sentence-per-line formatting like most of the rest of the developer documentation.